### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by EevanR**
Branch: styling

- css updates
- changed functionality to hide display buttons based on signed in or not
- change feature test, due to css
 - minor functionality updates

Screen Shot 2019-12-15 at 9 43 52 PM

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -1,15 +1,76 @@
...
+  background-color: rgba(232, 228, 228, 0.954);
+}
+
+.addedFoodDiv {
+  background-color: rgb(250, 237, 237);
+  padding-top: 20px;
+  border-radius: 15px;
+}
+
+.totalprice {
```
Comment (mantanquzzmo) [inline app/assets/stylesheets/application.css]: should be camelcase

File: app/views/products/index.html.erb [hunk 2]
```diff
@@ -2,18 +2,24 @@
...
         <ul>
-          <li><%= product.description %></li>
-          <li><%= product.price %> SEK</li>
-          <%= button_to 'Add to order', order_items_path(id: product.id) %>
+          <li class="innerCard" id=<%= dom_id(product) %>><%= product.name %>
+            <ul>
+              <li><%= product.description %></li>
+              <li><%= product.price %> SEK</li>
+              <% if user_signed_in? %>
+              <%= button_to 'Add to order', order_items_path(id: product.id) %>
```
Comment (mantanquzzmo) [inline app/views/products/index.html.erb]: indentation

File: app/views/layouts/_nav.html.erb [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+<nav style="position: fixed; width: 100%; top; 0;"
+  class="navbar navbar-expand-md navbar-dark bg-dark">
+    <div class="navbar-collapse collapse w-100 order-1 order-md-0 dual-collapse2">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-link"><%= link_to 'Menu', root_path, class: 'navbar-brand' %><li>
+          <li class="nav-link">
+          <% if user_signed_in? %>
+            <% else %>
+            <%= link_to 'Login', new_user_session_path, class: 'navbar-brand' %><li>
+            <% end %>
```
Comment (mantanquzzmo) [inline app/views/layouts/_nav.html.erb]: indentation

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (EevanR) [reply app/assets/stylesheets/application.css]: done

File: app/views/products/index.html.erb (hunk 2 above)
Comment (EevanR) [reply app/views/products/index.html.erb]: done

File: app/views/layouts/_nav.html.erb (hunk 3 above)
Comment (EevanR) [reply app/views/layouts/_nav.html.erb]: done

**Pull request merged**